doc/man-index.rst

```diff
@@ -56,6 +56,7 @@ documented elsewhere.
     * :program:`ana_scans.py`
     * :program:`anaSBitTresh.py`
     * :program:`anaUltraLatency.py`
+    * :program:`anaDACScan.py`
```

Comment:
This will add `anaDACScan.py` to the list of undocumented tools. Since you wrote some documentation (great!), I'd add to "Analysis tools" toctree (as `man/anaDACScan`). I case you wonder, you can find a rendering of the corresponding page [here](http://homepage.iihe.ac.be/~lmoureau/GEM/sphinx/html/man-index.html) (unfortunately I don't update it very often).